src/DayPickerInput.js

```diff
@@ -115,6 +115,7 @@ export default class DayPickerInput extends React.Component {
     }),
 
     onDayChange: PropTypes.func,
+    onClose: PropTypes.func,
```

Comment:
We should really call it `onDayPickerHide` since "hide" is the verb we use to hide the overlay and we have a method called `hideDayPicker`.